### PR TITLE
tikv: Documentation for `cert-allowed-cn` configuration directive

### PR DESCRIPTION
**Pull request by LINKIWI**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Added: documentation for existing configuration parameter [`cert-allowed-cn`](https://github.com/tikv/tikv/blob/36a32504806411b3d620d3abff06166cc7296484/components/security/src/lib.rs#L31) in TiKV configuration

This configuration parameter is implemented (and works as expected), but is missing from the documentation.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: N/A
- Other reference link(s): N/A

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by LINKIWI** [comment]
@dveeden Are there any other changes required to get this into a mergeable state? Thanks.

**Review by dveeden — APPROVED** [review]

**Review by hunterlxt — APPROVED** [review]

**Comment by LINKIWI** [comment]
@shichun-0415 @dveeden @en-jin19 Sorry for the ping; is this change ready to merge? Thanks

**Comment by shichun-0415** [comment]
> @shichun-0415 @dveeden @en-jin19 Sorry for the ping; is this change ready to merge? Thanks

Yes. It seems it is ready to merge. @TomShawn PTAL. Thanks.

File: tikv-configuration-file.md [hunk 1]
```diff
@@ -1259,14 +1259,19 @@ Configuration items related to security
...
 ### `cert-path`
 
-+ The path of the Privacy Enhanced Mail (PEM) file that contains the X509 certificate
++ The path of the Privacy Enhanced Mail (PEM) file that contains the X.509 certificate
 + Default value: ""
 
 ### `key-path`
 
-+ The path of the PEM file that contains the X509 key
++ The path of the PEM file that contains the X.509 key
```
Comment (shichun-0415) [inline tikv-configuration-file.md:1267]:
```suggestion
+ The path of the PEM file that contains the X.509 key
```

**Comment by shichun-0415** [comment]
> @shichun-0415 @dveeden @en-jin19 Sorry for the ping; is this change ready to merge? Thanks

Yes. It seems it is ready to merge. @TomShawn PTAL. Thanks.

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**